VirtualReality/qSlicerVirtualRealityModuleWidget.cxx

```diff
@@ -116,11 +117,21 @@ void qSlicerVirtualRealityModuleWidget::updateWidgetFromMRML()
   d->DesiredUpdateRateSlider->setEnabled(vrViewNode != NULL);
   d->DesiredUpdateRateSlider->blockSignals(wasBlocked);
 
+  wasBlocked = d ->PhysicalScaleSlider->blockSignals(true);
+  d->PhysicalScaleSlider->setValue(vrViewNode != NULL ? vrViewNode->GetPhysicalScale() * 100.0 : 100.0 );
```

Comment:
Please use nullptr instead of NULL, it will be soon changed throughout Slicer core.

Reply:
Do want this to be change to be applied for all the buttons?

Reply:
Everwhere please. Not just all the buttons but all the classes you touched